src/Meziantou.Analyzer/Rules/DoNotRemoveOriginalExceptionFromThrowStatementAnalyzer.cs

```diff
@@ -11,8 +11,8 @@ public sealed class DoNotRemoveOriginalExceptionFromThrowStatementAnalyzer : Dia
 {
     private static readonly DiagnosticDescriptor s_rule = new(
         RuleIdentifiers.DoNotRemoveOriginalExceptionFromThrowStatement,
-        title: "Do not remove original exception",
-        messageFormat: "Do not remove original exception",
+        title: "Prefer rethrowing an exception implicitly",
```

Comment:
I'm not sure to understand why you say "implicitly". The rule force you to keep the original exception when throwing a new exception (exception wrapping). If you don't do it, there is no "implicit" exception, the original exception is lost.

Reply:
Isn't this rule equivalent to `CA2200: Rethrow to preserve stack details`? I based the description here on [that rule's example](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2200#example). The throw itself is not implicit, but the exception should be.